.github/fabricbot.json

```diff
@@ -8943,14 +8955,14 @@
                 {
                   "name": "titleContains",
                   "parameters": {
-                    "titlePattern": "az ad",
+                    "titlePattern": "az \\b(ad)\\b",
```

Comment:
Just out of curiosity, what does `\\b` mean here?

Reply:
`\\b` is `Word Boundaries` in regex.
In this case, you can match `ad` as a word, excluding the interference of `advisor` or `adxxx`.